src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -60,6 +62,12 @@ public function whenUserWillBeSaved(UserWillBeSaved $event)
             $this->assertCan($actor, 'suspend', $user);
 
             $user->suspend_until = new DateTime($attributes['suspendUntil']);
+
+            if (isset($attributes['suspendUntil'])) {
+                $user->raise(new UserWasSuspended($user, $actor));
+            } else {
+                $user->raise(new UserWasUnsuspended($user, $actor));
+            }
```

Comment:
This will cause conflicts with my latest changes on master. Please get rid of my ternary statement and instead integrate it into your if/else. :smile: